src/core/audio/AudioDevice.cpp

```diff
@@ -130,17 +130,18 @@ void AudioDevice::renamePort( AudioPort * )
 int AudioDevice::convertToS16(const SampleFrame* _ab,
 								const fpp_t _frames,
 								int_sample_t * _output_buffer,
-								const bool _convert_endian )
+								ch_cnt_t channels,
+								const bool _convert_endian)
 {
 	if( _convert_endian )
 	{
 		for( fpp_t frame = 0; frame < _frames; ++frame )
 		{
-			for( ch_cnt_t chnl = 0; chnl < channels(); ++chnl )
+			for( ch_cnt_t chnl = 0; chnl < channels; ++chnl )
```

Comment:
```suggestion
			for(ch_cnt_t chnl = 0; chnl < channels; ++chnl)
```

Do check at similar cases in this file